### PR TITLE
Add support to set map boundaries

### PR DESCRIPTION
**Pull request by codedre**
bound_example_ios
bound_example_android

Adds the ability to prevent scrolling outside of a set of boundary coordinates. Pass in two `LatLng` points, preferably a North East point and a South West point (In testing any two points worked and the overlapping area formed the desired region).

- Works on Android and Google Maps for iOS
- Solves #1424  (my issue 🎉 ) 
- [Android Docs](https://developers.google.com/maps/documentation/android-api/views#restricting_the_users_panning_to_a_given_area)
- [iOS Docs](https://developers.google.com/maps/documentation/ios-sdk/reference/interface_g_m_s_map_view.html#af18afcf5b4eab9edf2b700734d2c7f07)

### CODE REVIEW

**Comment by aaronbuchanan** [comment]
@8-uh

File: docs/mapview.md [hunk 1]
```diff
@@ -64,6 +64,7 @@ To access event data, you will need to use `e.nativeEvent`. For example, `onPres
 |---|---|---|
 | `animateToRegion` | `region: Region`, `duration: Number` |
 | `animateToCoordinate` | `coordinate: LatLng`, `duration: Number` |
+| `setMapBoundaries` | `northEast: LatLng`, `southWest: LatLng` |
```
Comment (christopherdro) [inline docs/mapview.md]: Maybe add a note here mentioning this prop is specific to google maps?

**Comment by aaronbuchanan** [comment]
Hoping to see this feature make it's way into a release soon. Anything I can do to help nudge along?

**Comment by prevuelta** [comment]
+1

**Comment by aaronbuchanan** [comment]
@christopherdro any sense on when this might land?

**Comment by aaronbuchanan** [comment]
@codedre you able to check out the conflicts that have come up since this was opened?

**Comment by aaronbuchanan** [comment]
I see that a couple of other recent PR merges that added lines around where `SET_MAP_BOUNDARIES` was being set to `6`, I changed it to:
```private static final int SET_MAP_BOUNDARIES = 8;```

Seems to work for me, but I did notice that zooming out allows me to get outside of the bounds

**Comment by aaronbuchanan** [comment]
Any thoughts on that last note @codedre ?

**Comment by juergengunz** [comment]
I get following error when building the project with this merged pull request:

> .../node_modules/react-native-maps/lib/ios/AirGoogleMaps/AIRGoogleMapManager.m:310:15: Property 'cameraTargetBounds' not found on object of type 'AIRGoogleMap *'

**Comment by nikolas7892** [comment]
Is there any example implementation for this? I tried to find the available method in the node modules but found none.. I've tried calling this method over and over with the provided documentation in the photo below... Really stumped here...
screen shot 2017-12-26 at 12 10 54 am

**Comment by EskelCz** [comment]
This is probably the issue
https://stackoverflow.com/questions/48138782/restrict-gmsmapview-to-certain-bounds

**Pull request merged**